sdk/communication/AzureCommunication/Tests/CommunicationIdentifierSerializerTests.swift

```diff
@@ -132,69 +189,25 @@ class CommunicationIdentifierSerializerTests: XCTestCase {
     }
 
     func test_SerializeMicrosoftTeamsUser_NotAnonymous_ExpectedIdString() throws {
-        try serializeMicrosoftTeamsUser(isAnonymous: false, expectedId: "some id")
+        try serializeMicrosoftTeamsUser(isAnonymous: false, expectedId: testRawId)
     }
 
     func test_SerializeMicrosoftTeamsUser_Anonymous_ExpectedIdString() throws {
-        try serializeMicrosoftTeamsUser(isAnonymous: true, expectedId: "some id")
+        try serializeMicrosoftTeamsUser(isAnonymous: true, expectedId: testRawId)
     }
 
     func serializeMicrosoftTeamsUser(isAnonymous: Bool, expectedId: String?) throws {
-        let model = try CommunicationIdentifierSerializer
-            .serialize(identifier: MicrosoftTeamsUserIdentifier(
-                userId: "user id",
-                isAnonymous: isAnonymous,
-                rawId: expectedId
-            ))
-
-        XCTAssertEqual(model.kind, .microsoftTeamsUser)
-        XCTAssertEqual(model.microsoftTeamsUserId, "user id")
-        XCTAssertEqual(model.isAnonymous, isAnonymous)
-        XCTAssertEqual(model.id, expectedId)
-    }
-
-    func test_DeserializeMissingProperty() throws {
```

Comment:
why is this test not needed anymore?

Reply:
<img width="675" alt="Screen Shot 2021-02-11 at 5 42 46 PM" src="https://user-images.githubusercontent.com/9044372/107721084-98d33100-6c90-11eb-8ebe-3c2d86c95c4c.png">
Since the new init has a bunch of default parameters technically these tests for the model become less relevant/ not possible.

Reply:
but some of the combinations that we want to prohibit are valid in the schema. For example, in deserialization we want to enforce `rawId` to be present. In this example `rawId` is nullable because it is not needed for serialization.

Reply:
Oh I see your point. Ok yes I will need to update these tests!

Reply:
@RezaJooyandeh re-introduced the these tests in the `Chat/IdentifierSerializerTests`